client/lib/plans/plans-list.js

```diff
@@ -1546,7 +1546,11 @@ export const PLANS_LIST = {
 
 		// TODO: no idea about this, copied from the WP.com Premium plan.
 		// Features not displayed but used for checking plan abilities
-		getHiddenFeatures: () => [ constants.FEATURE_AUDIO_UPLOADS ],
+		getHiddenFeatures: () => [
+			constants.FEATURE_AUDIO_UPLOADS,
+			constants.FEATURE_JETPACK_SEARCH,
```

Comment:
This is for the new P2+ plan. This ensures that Jetpack Search settings are available for sites on this plan.